test/test.js

```diff
@@ -1,12 +1,12 @@
 import fs from 'fs';
 import {spawn} from 'child_process';
+import execa from 'execa';
 import test from 'ava';
 import pathExists from 'path-exists';
-import execFile from 'get-exec-file';
 import {version as pkgVersion} from '../package.json';
 
```

Comment:
Add:

`process.chdir(__dirname);`

As we plan on changing the behavior in AVA.